GAE/src/org/waterforpeople/mapping/dataexport/RawDataSpreadsheetImporter.java

```diff
@@ -428,7 +428,7 @@ public InstanceData parseInstance(Sheet sheet, int startRow,
 
                         case DATE:
                             String dateString = ExportImportUtils.parseCellAsString(cell);
-                            Date date = ExportImportUtils.parseDate(dateString);
+                            Date date = ExportImportUtils.parseSpreadsheetDate(dateString);
                             if (date != null) {
                                 val = String.valueOf(date.getTime());
```

Comment:
Just realised that incase it happens that the format here is the old one i.e. `dd-MM-yyyy HH:mm:ss Z`, we should set the timestamp to 00h00 that day.